addons/project/static/tests/project_subtask_kanban_list_tests.js

```diff
@@ -24,12 +24,13 @@ QUnit.module('Subtask Kanban List tests', {
             { name: "User One", login: 'one', password: 'one' },
         ]);
         pyEnv['project.task'].create([
-            { name: 'task one', project_id: projectId, closed_subtask_count: 2, subtask_count: 3, child_ids: [2, 3, 4], state: '01_in_progress', user_ids: [userId] },
+            { name: 'task one', project_id: projectId, closed_subtask_count: 1, subtask_count: 4, child_ids: [2, 3, 4, 7], state: '01_in_progress', user_ids: [userId] },
             { name: 'task two', parent_id: 1, closed_subtask_count: 0, subtask_count: 0, child_ids: [], state: '03_approved' },
             { name: 'task three', parent_id: 1, closed_subtask_count: 0, subtask_count: 0, child_ids: [], state: '02_changes_requested' },
             { name: 'task four', parent_id: 1, closed_subtask_count: 0, subtask_count: 0, child_ids: [], state: '1_done' },
             { name: 'task five', closed_subtask_count: 0, subtask_count: 1, child_ids: [6], state: '03_approved' },
             { name: 'task six', parent_id: 5, closed_subtask_count: 0, subtask_count: 0, child_ids: [], state: '1_canceled' },
+            { name: 'task one', parent_id: 1, closed_subtask_count: 0, subtask_count: 0, child_ids: [], state: '01_in_progress', user_ids: [userId] },
```

Comment:
Is there a reason why this has the same name as the task of line 27 ? 
Changing it would be clearer imo (the name scheme that I originally chose wasn't the best admittedly)

Reply:
No, there isn't, in fact, I copied and pasted and I forgot to change the name. 🙈